### PR TITLE
Add release notes for v1.5.0

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by mheon** [comment]
@rhatdan @giuseppe Do we want to include something about the storage flag to ignore chown errors?

**Comment by mheon** [comment]
/hold

Reviews are welcome, but let's not merge until all other PRs tagged for 1.5 are in - if some don't make it, we may need to adjust the notes.

**Comment by rhatdan** [comment]
Yes lets add something about the ignorechownerrors, seeing it is for potential HPC Use Cases.  We can say it is for experimental use cases.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
 # Release Notes
 
+## 1.5.0
+### Features
+- Podman containers can now join the user namespaces of other containers with `--userns=container:$ID`, or a user namespace at an arbitary path with `--userns=ns:$PATH`
+- Rootless Podman can experimentally be run with only a single UID and GID (without the `newuidmap` and `newgidmap` executables) by passing `--storage-opt ignore_chown_errors`
```
Comment (rhatdan) [inline RELEASE_NOTES.md]: Add `Even if the image contains more then one UID.`

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+- Added the ability to use `podman import` with URLs ([#3609](https://github.com/containers/libpod/issues/3609))
+- The `podman ps` command now supports filtering names using regular expressions ([#3394](https://github.com/containers/libpod/issues/3394))
+- Rootless Podman containers with `--privileged` set will now mount in all host devices that the user can access
+- The `podman create` and `podman run` commands now support the `--env-host` flag to forward all environment variables from the host into the container
+- Rootless Podman now supports healthchecks ([#3523](https://github.com/containers/libpod/issues/3523))
+- The format of the `HostConfig` portion of the output of `podman inspect` on containers has been improved and synced with Docker
+- Podman containers now support CGroup namespaces, and can create them by passing `--cgroupns=private` to `podman run` or `podman create`
+- The `podman create` and `podman run` commands now support the `--ulimit=host` flag, which uses any ulimits currently set on the host for the container
+- The `podman rm` and `podman rmi` commands now use different exit codes to indicate 'no such container' and 'container is running' errors
+
```
Comment (rhatdan) [inline RELEASE_NOTES.md]: Should we  mention something about cgroupsV2 and crun?  @giuseppe WDYT?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,67 @@
...
+### Misc
+- Updated vendored Buildah to v1.10.1
+- Updated vendored containers/image to v3.0.2
+- Updated vendored containers/storage to v1.13.1
+- Podman now requires conmon v1.0.0-rc2 or higher
+- The `podman info` command now displays the events logger being in use
+- The `podman inspect` command on containers now includes the ID of the pod a container has joined and the PID of the container's conmon process
+- The `-v` short flag for `podman --version` has been re-added
+- Error messages from `podman pull` should be significantly clearer
+- The `podman exec` command is now available in the remote client
```
Comment (haircommander) [inline RELEASE_NOTES.md]: should probably mention a new conmon version is required

Comment (mheon) [reply RELEASE_NOTES.md]: I believe that's in there - in Misc?

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,67 @@
...
+- Fixed a bug where Podman would unconditionally append `ENTRYPOINT` to `CMD` during `podman commit` (and when reporting `CMD` in `podman inspect`) ([#3708](https://github.com/containers/libpod/issues/3708))
+- Fixed a bug where `podman events` with the `journald` events backend would incorrectly print 6 previous events when only new events were requested ([#3616](https://github.com/containers/libpod/issues/3616))
+- Fixed a bug where `podman port` would exit prematurely when a port number was specified ([#3747](https://github.com/containers/libpod/issues/3747))
+- Fixed a bug where passing `.` as an argument to the `--dns-search` flag to `podman create` and `podman run` was not properly clearing DNS search domains in the container
+
+### Misc
+- Updated vendored Buildah to v1.10.1
+- Updated vendored containers/image to v3.0.2
+- Updated vendored containers/storage to v1.13.1
+- Podman now requires conmon v1.0.0-rc2 or higher
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: still hold?

Comment (mheon) [reply RELEASE_NOTES.md]: Yep, even if we have 2.0, v1.0.0-rc2 tag will work

**Comment by TomSweeneyRedHat** [comment]
Other than @rhatdan's comments, LGTM

File: RELEASE_NOTES.md (hunk 4 above)
Comment (haircommander) [reply RELEASE_NOTES.md]: I would say this should be v2.0.0, even though it's not yet enforced by the code

Comment (mheon) [reply RELEASE_NOTES.md]: Done

**Comment by mheon** [comment]
All comments addressed

**Pull request merged**